src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -15,14 +29,18 @@ function print_info_from_core_file {
     return
   fi
 
+  print_info_from_core_file_using_lldb $core_file_name $executable_name
+
   # Check for the existence of GDB on the path
   hash gdb 2>/dev/null || { echo >&2 "GDB was not found. Unable to print core file."; return; }
```

Comment:
I suggest you now extract the gdb work into a very similar function so code is consistent